charts/stable/standardnotes-server/values.yaml

```diff
@@ -0,0 +1,285 @@
+image:
+  pullPolicy: IfNotPresent
+  repository: standardnotes/server
+  tag: latest@sha256:cbdc5e3e6d7212eeaec86898e6ba944b1678349262d3da7215a246d832471391
+localstackImage:
+  pullPolicy: IfNotPresent
+  repository: localstack/localstack
+  tag: 3.0@sha256:73698e485240939490134aadd7e429ac87ff068cd5ad09f5de8ccb76727c13e1
+persistence:
+  logs:
+    enabled: true
+    targetSelector:
+      main:
+        main:
+          mountPath: /var/lib/server/logs
+  uploads:
+    enabled: true
+    targetSelector:
+      main:
+        main:
+          mountPath: /opt/server/packages/files/dist/uploads
+  localstack-bootstrap:
+    enabled: true
+    type: configmap
+    objectName: localstack-bootstrap-init
+    defaultMode: "0777"
+    targetSelector:
+      localstack:
+        localstack:
+          mountPath: /etc/localstack/init/ready.d/localstack_bootstrap.sh
+          subPath: localstack_bootstrap.sh
+portal:
+  open:
+    enabled: false
+securityContext:
+  container:
+    readOnlyRootFilesystem: false
+    runAsGroup: 0
+    runAsUser: 0
+service:
+  main:
+    enabled: true
+    ports:
+      main:
+        enabled: true
+        port: 3000
+        targetPort: 3000
+      files:
+        enabled: true
+        port: 3125
+        targetPort: 3104
+  localstack:
+    enabled: true
+    targetSelector: localstack
+    ports:
+      app:
+        enabled: true
+        targetSelector: localstack
+        port: 4566
+        targetPort: 4566
+workload:
+  main:
+    podSpec:
+      containers:
+        main:
+          env:
+            DB_HOST:
+              secretKeyRef:
+                expandObjectName: false
+                name: '{{ printf "%s-%s" .Release.Name "mariadbcreds" }}'
+                key: plainhost
+            DB_PORT: 3306
+            DB_USERNAME: std_notes_user
+            DB_PASSWORD:
+              secretKeyRef:
+                expandObjectName: false
+                name: '{{ printf "%s-%s" .Release.Name "mariadbcreds" }}'
+                key: mariadb-password
+            DB_DATABASE: standard_notes_db
+            DB_TYPE: mysql
```

Comment:
I suppose there is no PG support?

Reply:
No pg support. It looks like the container supports mysql by default with an optional sqlite driver.
https://github.com/standardnotes/server/blob/5c9f493b67a65aa9e659db77e797635b8aa09fc3/.github/workflows/e2e-home-server.yml#L21